lib/features/home_page.dart

```diff
@@ -8,6 +8,7 @@ class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     StoreProvider.dispatch(context, GetPokemonsAction());

```

Comment:
add a todo comment that these will be removed later 🙂

Reply:
resolved